cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -46,4 +46,27 @@ function up() {
             $kubectl wait networkaddonsconfig cluster --for condition=Available --timeout=200s
         fi
     fi
+
+    if [ "$KUBEVIRT_DEPLOY_ISTIO" == "true" ]; then
+        if [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
+            $kubectl create -f /opt/istio/istio-operator-with-cnao.cr.yaml
+        else
+            $kubectl create -f /opt/istio/istio-operator.cr.yaml
+        fi
+        
+        istio_cni_ns=kube-system
+        istio_cni_ds_name=istio-cni-node
+        retries=0
+        while [[ $retries -lt 20 ]]; do
+            echo "waiting for istio-cni-node pods to be ready"
+            sleep 5
+            retries=$((retries + 1))
+            $kubectl -n $istio_cni_ns get daemonset $istio_cni_ds_name || continue
+            ready=$(kubectl -n $istio_cni_ns get daemonset $istio_cni_ds_name -o jsonpath="{.status.numberReady}")
+            required=$(kubectl -n $istio_cni_ns get daemonset $istio_cni_ds_name -o jsonpath="{.status.desiredNumberScheduled}")
+            if [[ $ready -eq $required ]]; then
+                break
+            fi
+        done
+    fi
```

Comment:
We should rely on IstioOperator status https://istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#InstallStatus